packages/compat/plugin-swc/src/utils.ts

```diff
@@ -184,6 +196,24 @@ const getCoreJsVersion = (corejsPkgPath: string) => {
   }
 };
 
+const reduceTransformImportConfig = (
+  options: NormalizedSourceConfig['transformImport'],
+): TransformImport[] | false => {
+  if (options === false || !options) {
+    return false;
+  }
+
+  let imports: TransformImport[] = [];
+  for (const item of castArray(options)) {
+    if (isFunction(item)) {
```

Comment:
Seems no need to create a `isFunction` helper here:

```suggestion
    if (typeof item === 'function') {
```